process/divertor.py

```
@@ -61,692 +61,6 @@ def run(self, output: bool) -> None:
             )
             return
 
```

Maybe raise an error here if none of the options are selected

---

Have set it so that `divtart()` will run with just the `i_hldiv = 1` condition. Both models are applicable to ST's though `i_hldiv = 2` is more advanced